src/shared/seccomp-util.c

```diff
@@ -1112,6 +1112,17 @@ int seccomp_memory_deny_write_execute(void) {
                 r = seccomp_rule_add_exact(
                                 seccomp,
                                 SCMP_ACT_ERRNO(EPERM),
+                                SCMP_SYS(mmap2),
```

Comment:
what do you mean by "doesn't make sense"?

I mean, my patch always also blocks mmap2, and libseccomp figures out the reset, and does the right thing depending on whether the arch has the syscall or not...